lib/project_types/rails/gem.rb

```diff
@@ -34,18 +34,20 @@ def gem_path(ctx)
       private
 
       def apply_gem_home(ctx)
-        path = File.join(ctx.getenv('HOME'), '.gem', 'ruby', RUBY_VERSION)
+        path = ''
```

Comment:
I think this could be a little hard to make sense of without reading PR description, is it perhaps worth some adding some explanatory comments to give additional context?